client/tests/webdriver/baseSpecs/createReportWithPlanningConflict.spec.js

```diff
@@ -162,8 +162,9 @@ describe("When creating a Report with conflicts", () => {
     expect(ShowReport.reportConflictIcon.isExisting()).to.equal(true)
 
     ShowReport.reportConflictIcon.moveTo()
-    expect(ShowReport.reportConflictTooltipTitle).to.equal(
-      "3 of 5 attendees are busy at the selected time!"
+    // Depending on the order of the tests, the number of conflicts may vary
```

Comment:
IMHO, separating the test data to get consistent results would be a better solution than accepting both results.

Reply:
Agreed, but that is potentially a _lot_ of work, and this is intended to be a quick fix in the interim (saving us from having to rerun build jobs).